src/commands/scanner/rule/describe.ts

```diff
@@ -50,23 +54,37 @@ export default class Describe extends ScannerCommand {
 			const msg = messages.getMessage('output.multipleMatchingRules', [rules.length.toString(), this.flags.rulename]);
 			this.ux.warn(msg);
 			rules.forEach((rule, idx) => {
-				this.ux.styledHeader('Rule #' + (idx + 1));
+				this.ux.styledHeader(`Rule #${idx + 1}`);
 				this.logStyledRule(rule);
 			});
 		} else {
+			// If there's exactly one rule, we don't need to do anything special, and can just log the rule.
 			this.logStyledRule(rules[0]);
 		}
 		// We need to return something for when the --json flag is used, so we'll just return the list of rules.
 		return JSON.parse(JSON.stringify(rules));
 	}
 
-	private static formatRuleForDescribe(rule: Rule): object {
-		// Strip any whitespace off of the description.
-		rule.description = rule.description.trim();
-		return rule;
+	private async styleRules(rules: Rule[]): Promise<DescribeStyledRule[]> {
+		const allEngines = await Controller.getAllEngines();
```

Comment:
Is there a reason not to use the existing `Controller#getEnabledEngines`? If so, could you add a comment.